src/types.rs

```diff
@@ -138,7 +147,7 @@ pub type Borrowed<'e, T> = &'e Shared<T>;
 #[derive(Clone)]
 pub struct RNode {
     tree: Shared<Tree>,
-    inner: Node<'static>,
+    pub(crate) inner: Node<'static>,
```

Comment:
Accessing this is generally unsafe, because of the lying lifetime. It's better to use `rnode.borrow()` (which dereferences to `Node`), so that the burden of safety proof is on this sub-module, not the whole crate, unless we are sure it's a performance issue.

Reply:
In earlier code I've had some borrowing problems, I forgot to change them back. Thanks.